Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_First_SSO_Access.yml

```diff
@@ -0,0 +1,1906 @@
+        - Indicators of compromise (IOCs) related to the investigation.
+        - Attack patterns related to the investigation.
+        - Campaigns related to the investigation.
+        - IOCs associated with the identified campaigns.
+        - Reports containing details on the identified campaigns.
+      playbookName: TIM - Indicator Relationships Analysis
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "158"
+    scriptarguments:
+      Indicator:
+        complex:
+          root: inputs.IPAddress
+      LimitResults:
+        simple: "200"
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 0
+    view: |-
+      {
+        "position": {
+          "x": 707.5,
+          "y": 20
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "150":
+    id: "150"
+    taskid: 30e19b73-6556-4df0-8440-79122e149c1f
+    type: regular
+    task:
+      id: 30e19b73-6556-4df0-8440-79122e149c1f
+      version: -1
+      name: Set Related Campaign
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "6"
+    scriptarguments:
+      key:
+        simple: PartOfCampaign
+      value:
+        complex:
+          root: RelatedCampaign
+          filters:
+          - - operator: isExists
+              left:
+                value:
+                  simple: RelatedCampaign
+                iscontext: true
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 700,
+          "y": 380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    fieldMapping:
+    - incidentfield: Part of Campaign
+      output:
+        complex:
+          root: RelatedCampaign
+    - incidentfield: Related Report
+      output:
+        complex:
+          root: RelatedReport
+    skipunavailable: true
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "151":
+    id: "151"
+    taskid: f2feaf4b-662c-4cc1-8558-33ae9f305b2c
+    type: playbook
+    task:
+      id: f2feaf4b-662c-4cc1-8558-33ae9f305b2c
+      version: -1
+      name: User Investigation - Generic
+      description: |-
+        This playbook performs an investigation on a specific user, using queries and logs from SIEM, identity management systems, XDR, and firewalls.
+
+        Supported integrations:
+        -Okta
+        -Splunk
+        -QRadar
+        -Azure Log Analytics
+        -PAN-OS
+        -XDR By Palo Alto Networks
+      playbookName: User Investigation - Generic
```

Comment:
All the inputs from this playbook should be at the main one as well.

Reply:
Done.